### PR TITLE
update to latest pako 0.2.6 and improve readme

### PR DESCRIPTION
**Pull request by hipitihop**

### CODE REVIEW

File: pako/README.md [hunk 1]
```diff
@@ -12,5 +12,24 @@ you can require the packaged library like so:
 
 ```clojure
 (ns application.core
-  (:require cljsjs.pako))
+  (:require cljsjs.pako :as pako))
```
Comment (martinklepsch) [inline pako/README.md]: When requiring foreign libs using `:as` or `:refer` is not supported. Functionality will always be imported into the global namespace the same way as the regular library makes itself available. In most cases this means `js/jQuery`, `js/pako` etc.

**Comment by martinklepsch** [comment]
Besides the one comment I left this looks good. :+1:

File: pako/README.md (hunk 1 above)
Comment (hipitihop) [inline pako/README.md]: One learns something everyday, hopefully, thanks :-)
So do I need to do anything more or have you corrected this already and will merge ?

**Comment by martinklepsch** [comment]
I made the change and merged it https://github.com/cljsjs/packages/commit/465ce4050de0949b9a3f57b99a6c618437c0c5a4

**Pull request closed without merging**